operators/cloud-native-postgresql/1.24.0/manifests/cloud-native-postgresql.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1308 @@
+              resources:
+                - scheduledbackups
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - postgresql.k8s.enterprisedb.io
+              resources:
+                - scheduledbackups/status
+              verbs:
+                - get
+                - patch
+                - update
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - rolebindings
+              verbs:
+                - create
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - roles
+              verbs:
+                - create
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - snapshot.storage.k8s.io
+              resources:
+                - volumesnapshots
+              verbs:
+                - create
+                - get
+                - list
+                - patch
+                - watch
+          serviceAccountName: postgresql-operator-manager
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: true
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - postgresql
+    - postgres
+    - database
+    - sql
+    - cloudnativepg
+    - cloudnative-pg
+    - cnp
+  links:
+    - name: EDB Postgres for Kubernetes
+      url: https://www.enterprisedb.com/products/postgresql-on-kubernetes-ha-clusters-k8s-containers-scalable
+    - name: Documentation
+      url: https://www.enterprisedb.com/docs/postgres_for_kubernetes/latest/
+  maintainers:
+    - email: jonathan.gonzalez@enterprisedb.com
+      name: Jonathan Gonzalez V.
+    - email: john.long@enterprisedb.com
+      name: John Long
+    - email: gabriele.bartolini@enterprisedb.com
+      name: Gabriele Bartolini
+  maturity: stable
+  minKubeVersion: 1.23.0
+  provider:
+    name: EDB
+  relatedImages:
+    - image: quay.io/enterprisedb/cloud-native-postgresql@sha256:869112d57a886a573890e85a2798ef8157cc08babe7d149dbb512a97372226c1
+      name: cloud-native-postgresql-869112d57a886a573890e85a2798ef8157cc08babe7d149dbb512a97372226c1-annotation
+    - image: quay.io/enterprisedb/cloud-native-postgresql@sha256:869112d57a886a573890e85a2798ef8157cc08babe7d149dbb512a97372226c1
+      name: cnp
+    - image: quay.io/enterprisedb/cloud-native-postgresql@sha256:869112d57a886a573890e85a2798ef8157cc08babe7d149dbb512a97372226c1
+      name: manager
+  replaces: cloud-native-postgresql.v1.23.0
```

Comment:
This version was never released, your bot closed the PR.
https://github.com/redhat-openshift-ecosystem/certified-operators/pull/3789

But with that said, these operators use version specific channels, and replaces does not work across channels. If you have new versions of the operator targeting new versions of OCP, either remove the `replaces/skips` (assuming a new channel for a new minor version). Or if a new version of OCP is targeted for the same channel, the first version into the new OCP version catalog should use `skips` and any version thereafter can use `replaces` (assuming same channel).

Reply:
@acornett21  We're aware of it but we will not fix this, we're currently managing all the PRs in an internal case with Red Hat Partner programs to fix no one issue but all the issues that will appear :D Thanks for the help! if you can join on the ticket will be awesome too!

Reply:
I don't understand your reply, if you are aware that removing the line I mentioned would fix the issue, then why not fix the issue? You can't replace something that doesn't exist, and you can't replace something that's not in the same channel.

There is not anything for Red Hat to fix here.